app/models/labware_creators/pooled_tubes_by_sample.rb

```diff
@@ -0,0 +1,261 @@
+# frozen_string_literal: true
+
```

Comment:
As mentioned on Slack, this could do with a test, and possibly a refactor for some bits (think some bits are overcomplicated, for instance use of 'name_for' and 'name_for_details' methods). But again, we could get it onto UAT and then do this.